.github/workflows/ci.yml

```
@@ -39,6 +39,8 @@ jobs:
         with:
           toolchain: ${{ steps.parse-msrv.outputs.version }}
 
+      - uses: r7kamura/rust-problem-matchers@v1
```

```suggestion
      - uses: r7kamura/rust-problem-matchers@d58b70c4a13c4866d96436315da451d8106f8f08 @ v1
```

For most third-party actions, we pin the hash to lower the risk of a supply-chain attack. Please use the hash everywhere.

---

Dependabot will send PRs for bumping these.